modules/colossussspBidAdapter.js

```diff
@@ -63,6 +63,12 @@ export const spec = {
       'placements': placements
     };
 
+    if(bidderRequest){
```

Comment:
circleci is failing on a linting error from this line: [github-build-link](https://circleci.com/gh/HuddledMasses/Prebid.js/33?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Fix by changing line 66 to: `if (bidderRequest) {`

Reply:
done